paasta_tools/cli/cmds/spark_run.py

```diff
@@ -838,9 +859,8 @@ def configure_and_run_docker_container(
     if pod_template_path:
         volumes.append(f"{pod_template_path}:{pod_template_path}:rw")
 
-    volumes.append(
-        f"{system_paasta_config.get_spark_kubeconfig()}:{system_paasta_config.get_spark_kubeconfig()}:ro"
-    )
+    kubeconfig_dir = os.path.dirname(system_paasta_config.get_spark_kubeconfig())
+    volumes.append(f"{kubeconfig_dir}:{kubeconfig_dir}:ro")
```

Comment:
seems like maybe a bit of a footgun to rely on the OG filepath for both cases? they both use the same directory for now so it's probably fine (and we're likely not going to change that, but y'know :p)

Reply:
also, this could use a comment as to why we're mounting the directory (i.e., because the new kubeconfig relies on a helper script that will exist here)